src/components/DisplayTaxonName.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import { Text } from "components/styledComponents";
+import _ from "lodash";
+import type { Node } from "react";
+import React from "react";
+
+type Props = {
+  item: Object,
+};
+
+const DisplayTaxonName = ( { item: { user, taxon } }: Props ): Node => {
+  if ( !taxon ) {
+    const text = "unknown";
+    return <Text numberOfLines={1}>{text}</Text>;
+  }
+  const taxonData = {};
+
+  taxonData.rank = taxon.rank;
+
+  if ( taxon.preferred_common_name ) {
+    taxonData.commonName = _.trim( taxon.preferred_common_name );
+  }
+
+  let { name: sciName } = taxon;
```

Comment:
Could we add a comment here to remind ourselves that a lot of this logic mirrors the [SplitTaxon component](app/webpack/shared/components/split_taxon.jsx)? It might be helpful in case we need to make changes later.